src/main/java/org/geppetto/frontend/controllers/ConnectionHandler.java

```diff
@@ -190,6 +193,10 @@ public void loadGeppettoProject(String requestID, IGeppettoProject geppettoProje
 			setConnectionProject(geppettoProject);
 			websocketConnection.sendMessage(requestID, OutboundMessages.PROJECT_LOADED, projectJSON);
 
+			boolean persisted = !geppettoProject.isVolatile();
+			String update = "{\"persisted\":" + persisted + ",\"projectId\":" + geppettoProject.getId()  +"}";
```

Comment:
I notice you are building up an update message with string manipulation.  Is there an object that you can be calling methods on to do this instead?